hybridse/src/udf/udf.cc

```diff
@@ -88,6 +94,24 @@ int32_t year(int64_t ts) {
     return t.tm_year + 1900;
 }
 
+int32_t dayofyear(codec::Timestamp *ts) { return dayofyear(ts->ts_); }
+int32_t dayofyear(codec::Date *date) {
+    int32_t day, month, year;
+    if (!codec::Date::Decode(date->date_, &year, &month, &day)) {
+        return 0;
```

Comment:
Is there any case can cover those error blocks ?

Reply:
I added test cases to try to cover those blocks but the new test cases didn't get rid of the codecov warning. It is not obvious to me why this is happening, but I'll try figuring it out again later. If you know why the warning remains please let me know, thanks.

Reply:
Where is the warning?

Reply:
The warning is [here](https://github.com/4paradigm/OpenMLDB/pull/856/files/15f659029541ad68fd78ead491c2cdb12b5a87a2#diff-52d2571ba18bc7cccad0bf035f0f7c3405da30f48680b2796c010a0f7ed7f466)